Source/WPEFramework/GenericConfig.cmake

```diff
@@ -21,6 +21,8 @@ string(TOLOWER ${NAMESPACE} NAMESPACE_LIB)
 set(PORT 80 CACHE STRING "The port for the webinterface")
 set(BINDING "0.0.0.0" CACHE STRING "The binding interface")
 set(IDLE_TIME 180 CACHE STRING "Idle time")
+set(SOFT_KILL_CHECK 10  CACHE STRING "Soft kill check waiting time")
```

Comment:
May be adding _TIME at the end of both the configuration vars will help identify vars better?